scripts/build.sh

```diff
@@ -179,11 +179,13 @@ if [[ -n "${SANITIZERS:-}" ]]; then
   done
 fi
 
-# Clean the Derived Data between builds to help reduce flakiness.
-rm -rf ~/Library/Developer/Xcode/DerivedData
-
 case "$product-$method-$platform" in
   Firebase-xcodebuild-*)
+    # Coverage collection often cause retries to fail because of partial
+    # pre-existing data.
+    # TODO: Find a less blunt solution to this.
```

Comment:
Please use `TODO(name)` or `TODO(b/bug-id)` to give the reader some pointer toward how to figure out more about this.